nrf_rpc/CHANGELOG.rst

```diff
@@ -9,6 +9,12 @@ Changelog
 
 All the notable changes to this project are documented on this page.
 
+Main branch
+***********
+
+* Enabled to initialize the nRF RPC more than once.
```

Comment:
```suggestion
* Enabled the nRF RPC library to be initialized more than once.
```

Reply:
Once the final wording is established, we can remove the second redundant sentence.

Reply:
It is fine to me.